contribs/gmf/examples/search.js

```diff
@@ -0,0 +1,48 @@
+goog.provide('gmf-search');
+
+goog.require('gmf.mapDirective');
+goog.require('gmf.searchDirective');
+goog.require('ol.Map');
+goog.require('ol.View');
+goog.require('ol.layer.Tile');
+goog.require('ol.source.OSM');
+
+
+/** @const **/
+var app = {};
+
+
+/** @type {!angular.Module} **/
+app.module = angular.module('app', ['gmf']);
+
+
+
+/**
+ * @constructor
+ */
+app.MainController = function() {
+
+  proj4.defs('EPSG:21781',
+      '+proj=somerc +lat_0=46.95240555555556 +lon_0=7.439583333333333 +k_0=1 ' +
+      '+x_0=600000 +y_0=200000 +ellps=bessel ' +
+      '+towgs84=674.374,15.056,405.346,0,0,0,0 +units=m +no_defs');
```

Comment:
Ok. It's probably obvious but we should not use specific projections in the search directive.